### ISSUE TITLE
How can one POST application/json with this?

### ISSUE THREAD

**Issue by Panthro** [Telerik-Verified-Plugins/SecureHTTP#24]
I have been trying to use this plugin to post jsons to a REST api, I simply can't.

It always sends as application/x-www-form-urlencoded and tries to parse my json into params.

eg:

``` json
{
"id":1,
"name": "John Doe"
 }
```

becomes: `id=1&name=John+Doe`

**Comment by xtrembaker** [comment]
+1

**Comment by talamaska** [comment]
have you tried placing a correct header?

**Comment by xtrembaker** [comment]
Of course, at first, we we're using application/json as Header Content-Type, till we realize our API was refusing all request because it was receiving "urlencoded" data. The plugin do not care about the Content-Type we specify in the header, it always automatically transform payload as "urlencoded" data.
